okteto.yml

```diff
@@ -1,10 +1,15 @@
-name: hello-world
-image: okteto/hello-world:php-dev
-workdir: /app
-command: [ "php", "-S", "0.0.0.0:8080"]
-forward: 
-  - 8080:8080
-reverse:
-  - 9000:9000
-persistentVolume:
-  enabled: false
+build:
+  hello-world:
```

Comment:
we should also build `hello-world-dev`, from same `context` but for `target: dev`